pattern-2/bosh-release/jobs/wso2apim/templates/ctl.erb

```diff
@@ -19,7 +19,6 @@
 #
 # ----------------------------------------------------------------------------
 
-set +e
```

Comment:
Shouldn't the script stop if a command exits with a non-zero status?

Reply:
`set +e` is the default status. i.e this script does not exit even with a non-zero status. Therefore this is a redundant line.